Rakefile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'bundler'
```

Comment:
Probably, for the Rakefile and CI, we should take now what extensions do. Take [solidus_subscriptions](https://github.com/solidusio-contrib/solidus_subscriptions/blob/master/Rakefile) as an example. Do you think that's something doable here?

Reply:
I like the idea of rebuilding SolidusLegacyFrontend as a Solidus extension. It would make the support code consistent with the extensions we already have. I'll give it a try :+1:

Reply:
I rebuilt SolidusFrontend as an extension in the new PR update.